user/views.py

```diff
@@ -4,15 +4,17 @@
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.settings import api_settings
 
-from user.serializers import UserSerializer
+from user.serializers import UserSerializer, AuthTokenSerializer
 
 
 class CreateUserView(generics.CreateAPIView):
     serializer_class = UserSerializer
+    permission_classes = ()
```

Comment:
The `CreateUserView` currently has no permission classes set, allowing unrestricted access. Ensure this is intentional or consider adding appropriate permissions to secure user creation.